runtime/near-vm-runner/src/near_vm_runner/memory.rs

```diff
@@ -97,3 +97,7 @@ impl MemoryLike for NearVmMemory {
         Ok(unsafe { self.get_mut(offset, buffer.len())? }.copy_from_slice(buffer))
     }
 }
+
+impl Drop for NearVmMemory {
+    fn drop(&mut self) {}
+}
```

Comment:
What's the point of this drop impl? I'm thinking if it's required for something we should probably document it clearly here, because this will probably be removed by someone in the future as cleanup.

But I can't think of any reason why `NearVmMemory` would need an explicit `Drop` impl myself, so maybe we could just remove it?

Reply:
Ah, these are an artifact from the iteration I had that had `lazy_drop` taking `Box<dyn Drop>` heh.